chirp/drivers/wouxun.py

```diff
@@ -229,7 +229,7 @@ def get_prompts(cls):
 
     @classmethod
     def _get_querymodel(cls):
-        if isinstance(cls._querymodel, str):
+        if isinstance(cls._querymodel, (str, bytes)):
```

Comment:
I don't think this is the right solution, as `str` is never valid for this anymore. I think the best thing to do is just get rid of this whole thing and make `_querymodel` just always iterable (and also probably remove this `_get_querymodel()` thing entirely).

Would you mind making that change and testing with whatever you have?

Reply:
...alternately I can make the change, if you're willing to test. I have something that uses this model somewhere, but  it would take me a non-trivial amount of time to find it (and the cable for it).

Reply:
I kinda thought you'd suggest that as I was looking at this. :) 

Sure, I can make that change...but can only test on the kg-808g physically.

Reply:
Cool, thanks. I'll take responsibility for the other models :)

Reply:
Done and tested with my 805 radio.

Reply:
Thanks. Should we add a model alias mapping for the 805 to the 816? I want to squash these commits anyway, so I can do that when I squash if you think it's appropriate. Also, sounds like this should `Fixes #11208` not just relate to it right?

Reply:
So, I'm not sure the 805 matches the 816...I haven't written to the radio yet using chirp. I'd like to create a driver that supports all the settings...which will take a bit. 

I could copy the memory layout, change it a bit, and maybe stuff that in this PR without the settings as a first pass. The number of channels is 128 for the 805 and 199 for the 816....that might be the only change, but I've only JUST started looking at it. Thoughts?

If you think this fixes #11208, that's fine. However, that person is trying to read/write an 805, and as it stands...this code will corrupt their radio if they write past channel 128 using the 816 driver.

Reply:
Personally, I'd rather keep the fix to 816/818 drivers separate from the creation of a new driver for the 805.

Reply:
Okay, I just didn't know the details. Let's leave this as you have it, just fixing the `bytes` thing and cleaning up, related to that bug. Then keep actual support for the other radio as a separate thing. It's fine, btw, to subclass and disable features for the 805, we can merge that so people can at least use it for memories (if that's solid) and you can iterate on settings after. I really prefer smaller incremental patches anyway.

But, I want to squash these down. I'll do that now so we can merge. Thanks!